gatsby-node.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')
+const fetch = require('node-fetch')
+
+module.exports.createPages = async ({ reporter, actions }) => {
+  if (process.env.SECRET_PAGE_CONTENTS_URL) {
```

Comment:
I'd avoid secret -> protected or  guest only

VIctoria's secrets should be kept secret you know?